code/modules/jobs/job_types/captain.dm

```diff
@@ -15,6 +15,7 @@
 	exp_requirements = 1200
 	exp_type = EXP_TYPE_COMMAND
 	exp_type_department = EXP_TYPE_COMMAND
+	special_notice = "You are not a security officer. Do not do their job."
```

Comment:
This is really weird notice for a captain.

Reply:
> This is really weird notice for a captain.

Not really, a lot of Captains like to play as a validhunting Judge Jury and Executioner.